### PR TITLE
update-streams: include arch selection

### PR DESCRIPTION
**Pull request by miabbott**

### CODE REVIEW

File: modules/ROOT/pages/update-streams.adoc [hunk 1]
```diff
@@ -54,9 +54,11 @@ Switching between streams may introduce regressions or bugs due to skipping upda
 sudo systemctl stop zincati.service
 
 # Perform the rebase to a different stream
+# Supported architectures: aarch64, ppc64le, x86_64
```
Comment (bgilbert) [inline modules/ROOT/pages/update-streams.adoc:57]: We don't have ppc64le yet, right?

Comment (miabbott) [reply modules/ROOT/pages/update-streams.adoc:57]: Oops, got confused looking at the different refs on the remote

File: modules/ROOT/pages/update-streams.adoc [hunk 2]
```diff
@@ -54,9 +54,11 @@ Switching between streams may introduce regressions or bugs due to skipping upda
 sudo systemctl stop zincati.service
 
 # Perform the rebase to a different stream
+# Supported architectures: aarch64, x86_64
 # Available streams: "stable", "testing", and "next"
+ARCH="$(arch)"
```
Comment (dustymabe) [inline modules/ROOT/pages/update-streams.adoc:59]:
```suggestion
ARCH="x86_64"
```

I feel like `$(arch)` is more confusing. We should either switch it to a real value or switch `STREAM="testing"` below to `STREAM=$(stream)`.

Comment (miabbott) [reply modules/ROOT/pages/update-streams.adoc:59]: `$(arch)` is the output of the `arch` command...`$(stream) is going to result in an error

Comment (dustymabe) [reply modules/ROOT/pages/update-streams.adoc:59]: oh wow. TIL about `/usr/bin/arch`. Sorry for the noise.

**Review by dustymabe — APPROVED** [review]
LGTM

**Pull request merged**